source/support.tex

```diff
@@ -1380,26 +1380,45 @@
   using uint_least64_t = @\textit{unsigned integer type}@;
 
   using uintmax_t      = @\textit{unsigned integer type}@;
-  using uintptr_t      = @\textit{unsigned integer type}@; // optional
+  using uintptr_t      = @\textit{unsigned integer type}@;  // optional
 }
-\end{codeblock}
 
-\pnum
-The header also defines numerous macros of the form:
+#define INT[8, 16, 32, 64]_MIN @\seebelow@         // optional
+#define INT[8, 16, 32, 64]_MAX @\seebelow@         // optional
+#define UINT[8, 16, 32, 64]_MAX @\seebelow@        // optional
 
-\begin{codeblock}
-  INT_[FAST LEAST]{8 16 32 64}_MIN
-  [U]INT_[FAST LEAST]{8 16 32 64}_MAX
-  INT{MAX PTR}_MIN
-  [U]INT{MAX PTR}_MAX
-  {PTRDIFF SIG_ATOMIC WCHAR WINT}{_MAX _MIN}
-  SIZE_MAX
-\end{codeblock}
+#define INT_LEAST[8, 16, 32, 64]_MIN @\seebelow@
+#define INT_LEAST[8, 16, 32, 64]_MAX @\seebelow@
+#define UINT_LEAST[8, 16, 32, 64]_MAX @\seebelow@
 
-plus function macros of the form:
+#define INT_FAST[8, 16, 32, 64]_MIN @\seebelow@
```

Comment:
This seems worse: the never-defined pattern-matching language of "INT_[FAST LEAST]{8 16 32 64}_MIN" is both changed and used in what would otherwise be a preprocessor directive (rather than prose).